config/secrets.yml

```diff
@@ -57,6 +57,8 @@ default: &default
 
   show_pame_statistics: false
 
+  disallow_all_web_crawlers: <%= ENV["DISALLOW_ALL_WEB_CRAWLERS"] %>
```

Comment:
Dont forget to add the env to `.env.example` with an example of what needs to go in there